package.json

```diff
@@ -149,7 +149,9 @@
     "npm-check-updates": "^11.5.13",
     "pg": "^8.7.1",
     "pg-native": "^3.0.0",
-    "pino": "^6.13.3",
+    "pino": "^7.8.1",
+    "pino-v6": "npm:pino@^6.13.3",
```

Comment:
We might want to transform this into a pattern.

```
pino: x.y.z [LATEST]
pino-previous: x.y.z

graphql: x.y.z [LATEST]
graphql-previous: x.y.z
```

By this we can always test against the latest two major versions of a package.